core/services/ocr2/plugins/ocr2keeper/evm21/upkeepstate/store.go

```diff
@@ -119,8 +134,14 @@ func (u *upkeepStateStore) Start(pctx context.Context) error {
 					}
 
 					ticker.Reset(utils.WithJitter(u.cleanCadence))
+				case <-flushTicker.C:
+					u.flush(ctx)
+				case err := <-u.errCh:
```

Comment:
we write into `errCh` only from flush, let's also listen to it from there (or maybe avoid it, as we anyway just log the errors)

Reply:
I've just removed it!